tracer.spec

```diff
@@ -97,7 +85,7 @@ mkdir -p %{buildroot}/%{python3_sitelib}/tracer
 
 cp -a bin/tracer-2 %{buildroot}/%{_bindir}/
 cp -a bin/tracer-3 %{buildroot}/%{_bindir}/
-ln -s %{_bindir}/tracer-2 %{buildroot}/%{_bindir}/tracer
+ln -s %{_bindir}/tracer-3 %{buildroot}/%{_bindir}/tracer
```

Comment:
Well that is in contradiction with what the quote says. Also, please see follow up comment in that BZ:

> If you want to package only 1 version of the executable, it should be the Python 3 version. But because you want to package both Python 2 and Python 3 versions of the executable, the unversioned executable should be in the Python 2 subpackage. See this section:

followed by this link <http://python-rpm-porting.readthedocs.io/en/latest/tools.html#ported-rpm-spec-file>

Personally I would like to have binaries for both py2/py3 and unversioned. However if this approach is not correct and linked docs are not valid, please point me to right one.